aten/src/ATen/cuda/tunable/GemmCommon.h

```diff
@@ -81,7 +81,8 @@ struct GemmParams : OpParams {
   }
 
   std::string Signature() const override {
-    return c10::str(transa, transb, "_", m, "_", n, "_", k);
+    static std::string val = c10::str(transa, transb, "_", m, "_", n, "_", k);
```

Comment:
I find it cause incorrect signature after initializing the static val . It can't be modified after the initiation. 'static' must be removed.